Google.Api.Generator/ConfigProtos/common_resources.proto

```diff
@@ -0,0 +1,14 @@
+﻿syntax = "proto3";
```

Comment:
So will each language have their own proto here, because the common resources need to be specified with different info for each language?
I'm just wondering what that'll look like in github...

Reply:
C# is the only language that (so far) uses "common resource names", so this isn't (yet) an issue.
If any other language requires it, then potentially this definition could be extended.